src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2200,6 +2200,37 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
```

Comment:
Please put that in the existing `children` call instead of starting a second one. It will make things more consistent with existing code.